src/PresentationalComponents/CreateImageWizard/WizardStepReview.js

```diff
@@ -130,7 +130,7 @@ WizardStepReview.propTypes = {
     subscription: PropTypes.object,
     subscribeNow: PropTypes.bool,
     uploadAWSErrors: PropTypes.object,
-    subscriptionErrors: PropTypes.object,
+    isSubscriptionValid: PropTypes.object,
```

Comment:
Hm should this be `isValidSubscription`?

Reply:
You are totally correct. Thanks for catching this. Fixed now.